ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/SqlFunctions.scala

```diff
@@ -42,4 +43,17 @@ private[appendonlydao] object SqlFunctions {
     def arrayIntersectionValues(arrayColumn: String, parties: Set[Party]): String =
       s"array_intersection($arrayColumn, array[${format(parties)}])"
   }
+
+  //TODO need to implement
```

Comment:
i set appendonlydao case for oracle as not yet implemented rather than half-baked code with a todo - need to have something there as it is now a valid entry under DbType though